modules/data-entry/src/main/frontend/src/questionnaire/Form.jsx

```diff
@@ -349,9 +369,17 @@ function Form (props) {
             <Typography variant="overline">Entered by {data['jcr:createdBy']} on {moment(data['jcr:created']).format("dddd, MMMM Do YYYY")}</Typography>
             : ""
           }
+          {
+            data && data['jcr:isCheckedOut'] ?
+            <Typography variant="overline" className={classes.warningStatus}>Another user is editing</Typography>
+            : ""
```

Comment:
We should display "Another user is editing" only in edit mode.
```suggestion
            isEdit && data && data['jcr:isCheckedOut'] &&
            <Typography variant="overline" className={classes.warningStatus}>Another user is editing</Typography>
```

Reply:
Or we can leave it as is. As @IntegralProgrammer suggested, the warning message might dissuade users from clicking on "Edit" and thus there will be fewer conflicts.